src/components/Common/Module/FieldItem.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div
+    class="d-flex"
+  >
+    <font-awesome-icon
+      :icon="['fas', 'grip-vertical']"
+      class="align-baseline mt-1 mr-3 text-primary handle"
+    />
+    <b
+      class="text-dark cursor-default"
+    >
+      {{ fields }}
+    </b>
+    <small
+      v-if="field.isSystem"
+      class="pt-1 ml-auto cursor-default"
+    >
+      {{ $t('selector.systemField') }}
+    </small>
+    <font-awesome-icon
+      v-if="isSelected"
+      :icon="['far', 'eye']"
+      class="align-baseline mt-1 ml-auto text-muted pointer"
+      @click="$emit('unselect-field', field)"
+    />
+    <font-awesome-icon
+      v-else
+      :icon="['fas', 'eye-slash']"
+      class="align-baseline mt-1 ml-auto text-muted pointer"
+      @click="$emit('select-field', field)"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  i18nOptions: {
+    namespaces: 'field',
+  },
+
+  name: 'FieldItem',
+
+  props: {
+    field: {
+      type: Object,
+      required: true,
+    },
+
+    isSelected: {
+      type: Boolean,
+      required: true,
+    },
+  },
+
+  computed: {
+    fields () {
+      if (this.field.label) {
```

Comment:
Instead of straight up returning the solution you can instead create a variable that you manipulate.

let foo = ''

if (foo) -> for = 'bar'

at the end append * if is required 
`foo = `${foo} *`